### PR TITLE
Support for pull request labels

### PR DESCRIPTION
**Pull request by vlesierse**
As the documentation states it support to define a filter action on a label of a pull request. Currently this only works when it's an issue.

[Documentation](https://github.com/actions/bin/tree/master/filter#label) `Continue if the issue or pull request has the following label`

This pull request add the possibility to use the filter on pull requests as well.

### CODE REVIEW

File: filter/bin/label [hunk 1]
```diff
@@ -2,18 +2,23 @@
 
 set -e
 
-filter='select(.issue.labels!=null) | .issue.labels[].name'
+filter_issues='select(.issue.labels!=null) | .issue.labels[].name'
+filter_pullrequests='select(.pull_request.labels!=null) | .pull_request.labels[].name'
```
Comment (hfaulds) [inline filter/bin/label]:
```suggestion
filter="([] + .issue.labels + .pull_request.labels)[].name"
```
I had a play around with `jq` and I think we can keep the rest of the code the same as it was before if we switch to this query.
WDYT?

File: filter/test/fixtures/event.json [hunk 2]
```diff
@@ -14,6 +14,11 @@
     }
   },
   "pull_request": {
+    "labels": [
+      {
+        "name": "pr-urgent"
+      }
+    ],
```
Comment (hfaulds) [inline filter/test/fixtures/event.json]: **Suggestion** I have a preference for separate fixtures. This fixture isn't actually a valid response. I realise this isn't something we've always done in this repo but I think it's something that's worth moving towards.

Comment (vlesierse) [reply filter/test/fixtures/event.json]: Fixed it with a seperate fixture for a pull request. Thank you for the feedback.

**Comment by vlesierse** [comment]
Thank you!

**Comment by parkr** [comment]
Thank you for your contribution!

**Pull request merged**